MyPerfectOnboardingApplication/Sources/MyPerfectOnboarding.Contracts/Selectors/ExpressionExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Linq.Expressions;
+using System.Reflection;
+
+namespace MyPerfectOnboarding.Contracts.Selectors
+{
+    internal static class ExpressionExtensions
+    {
+        public static PropertyInfo ToPropertyInfo<TModel, TProperty>(this Expression<Func<TModel, TProperty>> propertySelector)
+        {
+            var member = propertySelector.Body as MemberExpression;
+            if (member == null)
```

Comment:
```suggestion
            if (!(propertySelector.Body is MemberExpression member))
```
There are these C# 7 features that R# suggests to you... E.g. [type pattern matching](https://docs.microsoft.com/en-us/dotnet/csharp/pattern-matching#the-is-type-pattern-expression) in this case :)